ios/RCCViewController.h

```diff
@@ -27,6 +27,8 @@ extern NSString* const RCCViewControllerCancelReactTouchesNotification;
 - (void)setStyleOnInit;
 - (void)updateStyle;
 - (void)setNavBarVisibilityChange:(BOOL)animated;
+- (void)setPreviewController:(RCCViewController *)controller;
```

Comment:
This should be implemented as a `@property`, rather than explicit getters and setters with private ivars.

Reply:
Thanks!